### PR TITLE
week2-homework

### PR DESCRIPTION
**Pull request by Saikiruthiga**
created routers for meals and reservations

### CODE REVIEW

File: api/src/routers/deleteReservation.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    if (!idValues.includes(String(id))) {
+      return response
+        .status(400)
+        .json({ Error: `No matching id found to delete the reservation` });
+    }
+    await knex("reservation").where({ id: id }).del();
+    response.json({ Message: "Successfully deleted" });
+  } catch (error) {
+    console.log(error);
+    response.status(500).json({ Error: "Internal server error" });
```
Comment (babak-f) [inline api/src/routers/deleteReservation.js:19]: Very good error handling 👍

File: api/src/routers/postMeal.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+const router = express.Router();
+router.post("/", async (request, response) => {
+  try {
+    const data = request.body;
+    if (Object.values(data).length === 0) {
+      return response
+        .status(404)
+        .json({ Error: "Please provide the meal details to create" });
+    }
+    const [newMealId] = await knex(`meal`).insert(data);
```
Comment (babak-f) [inline api/src/routers/postMeal.js]: It's better to use the simple string notation when you are not mixing in any variables.

**Review by babak-f — APPROVED** [review]
Great job and great error handling :)

File: api/src/routers/postMeal.js (hunk 2 above)
Comment (Saikiruthiga) [reply api/src/routers/postMeal.js]: I will keep this in my mind. Thanks for your review.

**Pull request merged**